modules/ROOT/pages/mtf-metadata-testing.adoc

```diff
@@ -0,0 +1,363 @@
+= Metadata 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Test the metadata defined for your module.
+
+== Before You Begin
+
+* Know how to define metadata in your module. For more information, refer to xref:metadata.adoc[].
+* To create metadata tests, add the `mtf-tools` dependency to your project:
++
+[source,xml,linenums]
+----
+<dependency>
+    <groupId>com.mulesoft.munit</groupId>
+    <artifactId>mtf-tools</artifactId>
+    <version>1.0.0</version>
+    <classifier>mule-plugin</classifier>
+    <scope>test</scope>
+</dependency>
+----
+
+CAUTION: Metadata testing is available since Mule runtime engine 4.2.0. When running your tests against an earlier version, suites containing metadata tests are ignored.
+
+== Tooling Test
+
+The `tooling-test` component validates the module's metadata at design time, unlike the `munit:test` component that validates the module's metadata at execution. The `tooling-test` component is represented as follows:
+
+[source,xml,linenums]
+----
+<mtf:tooling-test name="metadataTest">
+    ...
+</mtf:tooling-test>
+----
+
+When a suite with a tooling test runs, XML validations are disabled. When testing metadata, it might be necessary to avoid filling required parameters.
```

Comment:
What does this second sentence mean?
You might want to avoid testing required parameters?